shared/chat/inbox/row/build-team/container.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import * as Route from '../../../../actions/route-tree'
+import {teamsTab} from '../../../../constants/tabs'
+import {compose, connect, setDisplayName} from '../../../../util/container'
+import type {TypedState, Dispatch} from '../../../../util/container'
+import BuildTeam from '.'
+
+const mapStateToProps = (state: TypedState) => ({
+  showBuildATeam: state.teams.teamnames.size === 0,
+})
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  // Route to the teams tab and open the NewTeamDialog component
```

Comment:
Are you missing a key for the dialog here? this is routing to the root of the teams tab

Reply:
I revisited the designs and realized it should not open the newteamdialog